Local AI connection is not working

### Description

When using local AI there is an error during testing connection inside Velo

> error: Connection error.

AI features don't work as well inside the app

Logs from model in LMStudio:
```
2026-02-20 19:53:35 [DEBUG] Received request: OPTIONS to /v1/chat/completions
2026-02-20 19:53:35 [ERROR] [qwen2.5-14b-instruct] 'messages' field is required
```

**I'm testing using LMStudio**

Hope it will be easy fix, this may help:
```
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER] Supported endpoints:
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]   LM Studio API
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]    ->  GET  http://localhost:1234/api/v1/models
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]    ->  POST http://localhost:1234/api/v1/chat
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]    ->  POST http://localhost:1234/api/v1/models/load
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]    ->  POST http://localhost:1234/api/v1/models/download
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]    ->  GET http://localhost:1234/api/v1/models/download/status:job_id
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]   OpenAI-compatible
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]    ->  GET  http://localhost:1234/v1/models
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]    ->  POST http://localhost:1234/v1/responses
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]    ->  POST http://localhost:1234/v1/chat/completions
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]    ->  POST http://localhost:1234/v1/completions
2026-02-20 20:14:37  [INFO] [LM STUDIO SERVER]    ->  POST http://localhost:1234/v1/embeddings
```

### Steps to reproduce

Add local model in Velo settings

### Expected behavior

Testing connection inside Velo should work + AI features inside app should work with local models

### Operating system

Windows

### App version

0.4.9

### Account type

None

### Screenshots / logs

```
2026-02-20 19:53:35 [DEBUG] Received request: OPTIONS to /v1/chat/completions
2026-02-20 19:53:35 [ERROR] [qwen2.5-14b-instruct] 'messages' field is required
```

### Additional context

_No response_

--- a/src/services/ai/providers/ollamaProvider.test.ts
+++ b/src/services/ai/providers/ollamaProvider.test.ts
@@ -9,6 +9,10 @@ vi.mock("openai", () => {
   return { default: MockOpenAI };
 });
 
+vi.mock("@tauri-apps/plugin-http", () => ({
+  fetch: vi.fn(),
+}));
+
 import OpenAI from "openai";
 import { createOllamaProvider, clearOllamaProvider } from "./ollamaProvider";
 
@@ -25,7 +29,7 @@ describe("ollamaProvider", () => {
       expect(OpenAI).toHaveBeenCalledWith({
         baseURL: "http://localhost:11434/v1",
         apiKey: "ollama",
-        dangerouslyAllowBrowser: true,
+        fetch: expect.any(Function),
       });
     });
 
@@ -35,7 +39,7 @@ describe("ollamaProvider", () => {
       expect(OpenAI).toHaveBeenCalledWith({
         baseURL: "http://localhost:11434/v1",
         apiKey: "ollama",
-        dangerouslyAllowBrowser: true,
+        fetch: expect.any(Function),
       });
     });
   });

--- a/src/services/ai/providers/ollamaProvider.ts
+++ b/src/services/ai/providers/ollamaProvider.ts
@@ -1,4 +1,5 @@
 import OpenAI from "openai";
+import { fetch } from "@tauri-apps/plugin-http";
 import type { AiProviderClient, AiCompletionRequest } from "../types";
 
 let instance: OpenAI | null = null;
@@ -10,7 +11,7 @@ function getClient(serverUrl: string, model: string): OpenAI {
     instance = new OpenAI({
       baseURL: `${serverUrl.replace(/\/+$/, "")}/v1`,
       apiKey: "ollama",
-      dangerouslyAllowBrowser: true,
+      fetch,
     });
     cachedKey = cacheKey;
   }

